addons/binding/org.openhab.binding.boschindego/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: BoschIndego Binding
+Bundle-SymbolicName: org.openhab.binding.boschindego;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/bosch-indego-controller-lib-0.8.jar,
+ lib/httpclient-4.5.2.jar,
+ lib/httpcore-4.4.4.jar,
+ lib/commons-codec-1.9.jar,
```

Comment:
If not used, it can be removed. The http client will require the Java package "org.apache.commons.logging", so you should add a Package-Import for this. Then you can get rid off this library.